src/docker-outside-of-docker/README.md

```diff
@@ -66,7 +66,7 @@ services:
 
 - The defaults value `./` is added so that the `docker-compose.yaml` file can work when it is run outside of the container
 
-### Change the workspace to `${localWorkspaceFolder}`
+### 2. Change the workspace to `${localWorkspaceFolder}`
```

Comment:
Can we make these changes to https://github.com/devcontainers/features/blob/main/src/docker-outside-of-docker/NOTES.md#change-the-workspace-to-localworkspacefolder instead?